CHANGELOG.md

```diff
@@ -6,60 +6,62 @@ All notable changes to this project will be documented in this file. For info on
 
 ### Security
 
-- Do not use semicolon as GET parameter separator. ([#1733](https://github.com/rack/rack/pull/1733), [@jeremyevans](https://github.com/jeremyevans))
+- Do not use semicolon as GET parameter separator. ([#1733](https://github.com/rack/rack/pull/1733), [@jeremyevans])
 
 ### SPEC Changes
 
-- rack.input is no longer required to be rewindable.
-- rack.multithread/rack.multiprocess/rack.run_once are no longer required environment keys.
-- Response status must now be an integer greater than or equal to 100.
-- Response headers must now be an unfrozen hash.
+- `rack.input` is no longer required to be rewindable.
+- `rack.multithread/rack.multiprocess/rack.run_once` are no longer required environment keys.
+- Response `status` must now be an integer greater than or equal to 100.
+- Response `headers` must now be an unfrozen hash.
 - Response header keys can no longer include uppercase characters.
-- Response body can now respond to #call instead of #each, for the equivalent of response hijacking in previous versions.
-- Middleware must no longer call #each on the body, but they can call #to_ary on the body if it responds to #to_ary.
+- Response header values can be an `Array` to handle multiple values (and no longer supports `\n` encoded headers).
+- Response body can now respond to `#call` (streaming body) instead of `#each` (enumerable body), for the equivalent of response hijacking in previous versions.
+- Middleware must no longer call `#each` on the body, but they can call `#to_ary` on the body if it responds to `#to_ary`.
 
 ### Removed
 
-- Remove `rack.multithread`/`rack.multiprocess`/`rack.run_once`. These variables generally come too late to be useful. ([#1720](https://github.com/rack/rack/pull/1720), [@ioquatix](https://github.com/ioquatix), [@jeremyevans](https://github.com/jeremyevans)))
-- Remove deprecated Rack::Request::SCHEME_WHITELIST. ([@jeremyevans](https://github.com/jeremyevans))
+- Remove `rack.multithread`/`rack.multiprocess`/`rack.run_once`. These variables generally come too late to be useful. ([#1720](https://github.com/rack/rack/pull/1720), [@ioquatix], [@jeremyevans]))
+- Remove deprecated `Rack::Request::SCHEME_WHITELIST`. ([@jeremyevans])
 
 ### Added
 
-- `Rack::Headers` added to support lower-case header keys. ([@jeremyevans](https://github.com/jeremyevans))
-- `Rack::Utils#set_cookie_header` now supports `escape_key: false` to avoid key escaping.  ([@jeremyevans](https://github.com/jeremyevans))
+- `Rack::Headers` added to support lower-case header keys. ([@jeremyevans], [@ioquatix])
```

Comment:
I believe only people who wrote the related code for the CHANGELOG entry should be listed.  While you did propose the idea of enforcing lower case response header keys, as far as I am aware, you did not write any code related to `Rack::Headers`, so I think you should remove yourself from this line.

Reply:
Okay, fair enough. I did spend a huge amount of time on this proposal though and it was really something I cared about.

Reply:
```suggestion
- `Rack::Headers` added to support lower-case header keys. ([@jeremyevans])
```